src/api/endpoint/index.js

```diff
@@ -29,8 +29,8 @@ export default function(config: Config, auth: IAuth, storage: IStorageHandler) {
   // this might be too harsh, so ask if it causes trouble
   // $FlowFixMe
   app.param('package', validatePackage);
+  app.param('filename', validatePackage);
```

Comment:
Problem number 1: Scoped packages would have a `/` character here. Changing this to `validatePackage` resolves the 403.

Reply:
@rtsao if we remove `validateName` we are creating other issues allowing push forbid names to the registry.

Reply:
Oh ... I spoke too soon, if I did I land on `validateName` again. 🤔

Reply:
This might not be the best fix.

Alternatively, it might be better to make the `filename` parameter not include the scope?

FWIW, for these scoped packages, it seems: https://github.com/verdaccio/verdaccio/blob/master/src/api/endpoint/api/package.js#L74 this route handler is being used, not https://github.com/verdaccio/verdaccio/blob/master/src/api/endpoint/api/package.js#L68